compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/SymbolFactory.java

```diff
@@ -375,7 +364,7 @@ public ParameterSymbol createBallerinaParameter(BVarSymbol symbol, ParameterKind
         if (symbol == null) {
             return null;
         }
-        String name = symbol.getName().getValue().isBlank() ? null : symbol.getName().getValue();
+        String name = symbol.getOriginalName().getValue().isBlank() ? null : symbol.getOriginalName().getValue();
```

Comment:
Do we set empty string to the original name as well for this? 🤔  If we don't, we don't need this check.

Reply:
We may get an empty string